src/Form/MediaForm.php

```diff
@@ -5,16 +5,25 @@
 use Drupal\Component\Utility\NestedArray;
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\bene_media\MediaHelper as Helper;
+use Drupal\Core\Security\TrustedCallbackInterface;
```

Comment:
If I'm reading the documentation correctly, this object was introduced in Drupal 8.8: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Security%21TrustedCallbackInterface.php/interface/TrustedCallbackInterface/8.8.x

So let's update that in the module's .info file: https://github.com/thinkshout/bene_media/blob/8.x-2.x-bene/bene_media.info.yml#L7

Reply:
Perfect, thank you and merged!